compiler/codegen/OMRTreeEvaluator.hpp

```diff
@@ -57,10 +57,20 @@ class OMR_EXTENSIBLE TreeEvaluator
    static int32_t checkPositiveOrNegativePowerOfTwo(int32_t value);
    static int32_t checkPositiveOrNegativePowerOfTwo(int64_t value);
 
-   static TR::Register *compressedRefsEvaluator(TR::Node *node, TR::CodeGenerator *cg);
-   static TR::Register *computeCCEvaluator(TR::Node *node, TR::CodeGenerator *cg);
+   /**
+    * @brief unImpOpEvaluator A dummy evaluator for ops that could be supported
+    * by a codegen but that are not implemented
+    */
    static TR::Register *unImpOpEvaluator(TR::Node *node, TR::CodeGenerator *cg);
+
+   /**
+    * @brief badILOpEvaluator A dummy evaluator for ops that are not supprted by
+    * a codegen (e.g. because they are specific to another codegen)
```

Comment:
I suggest the example also include "...or are not applicable to a particular project)"